test/integration/init_int_test.go

```diff
@@ -219,6 +219,30 @@ func (suite *InitIntegrationTestSuite) TestInit_InferredOrg() {
 	suite.Contains(string(fileutils.ReadFileUnsafe(filepath.Join(ts.Dirs.Work, constants.ConfigFileName))), "ActiveState-CLI")
 }
 
+func (suite *InitIntegrationTestSuite) TestInit_ChangeSummary() {
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("cp.SendCtrlC() does not work on Windows")
+	}
+	suite.OnlyRunForTags(tagsuite.Init)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	project := "test-init-change-summary-" + sysinfo.OS().String()
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("init", "ActiveState-CLI/"+project, "--language", "python@3.10.10"),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Resolving Dependencies")
+	cp.Expect("Done")
+	ts.NotifyProjectCreated("ActiveState-CLI", project)
+	cp.Expect("Setting up the following dependencies:")
+	cp.Expect("└─ python@3.10.10")
+	cp.SendCtrlC() // don't care what happens after
+	cp.ExpectNotExitCode(0)
```

Comment:
Same as with checkout; would be good if we can assert only a single dependency was printed.